5_vizprep.yml

```diff
@@ -0,0 +1,42 @@
+target_default: 5_vizprep
+
+include:
+  - 2_process.yml
+
+packages:
+
+file_extensions:
+  - feather
+  - ind
+
+sources:
+  - 5_vizprep/src/prep_plot_funs.R
+
+targets:
+
+  5_vizprep:
```

Comment:
That's what I was imagining. Task table will generate dynamic targets that will depend on non dynamic targets in `5_vizprep.yml`

Reply:
I was thinking there would be a dynamic task table in `5_` for these time-specific functions. Bad idea?

Reply:
oh I see to create all the `storm_point_[date].rds`, etc.. in `5_vizprep/out`. I think that's probably good actually

Reply:
I advocated yesterday and am going to continue advocating for there to be just one task table, all in phase 6, to avoid funky dependencies across task tables.